v2/lib/load/index.js

```diff
@@ -75,12 +75,26 @@ module.exports = async function load(siteDir) {
   const context = {env, siteDir, siteConfig};
 
   // Initialize plugins.
-  const plugins = pluginConfigs.map(({name, options}) => {
-    // TODO: Resolve using node_modules as well.
-    // eslint-disable-next-line
-    const Plugin = require(path.resolve(__dirname, '../../plugins', name));
-    return new Plugin(options, context);
-  });
+  const pluginDir = path.resolve(__dirname, '../../plugins');
+  const plugins = pluginConfigs.map(
+    ({name, path: pluginPath = path.join(pluginDir, name), options}) => {
```

Comment:
Can we take this chunk out into a `loadPlugin`/`initializePlugin` method? Better for testing as well.

Reply:
I'll put it on next PR